changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 9.6.0 - xxxx-xx-xx =
+* Fix - Restricts the BNPLs promotional banner to only be displayed after version 9.7.0.
```

Comment:
Nit: we should remove the trailing `.` per our style guide

Reply:
Updated it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4411/commits/95ca3ee47a0743c8ed58deed0a2473f177f54313